src/Uno.UI/UI/Xaml/Shapes/Ellipse.iOSmacOS.cs

```diff
@@ -9,7 +9,9 @@ public partial class Ellipse : Shape
 	{
 		public Ellipse()
 		{
+#if __IOS__
 			ClipsToBounds = true;
```

Comment:
is there and equivalent for macOS ?

Reply:
Yeah as @dr1rrb  said the `ClipToBounds=true` is the default behavior for macOS. When needing the opposite we will have to come up with workarounds. Thankfully for the Shapes this was not necessary.